### ISSUE TITLE
New Evasion/Toughness mechanics and the cost/pre-requisites of the Natural Defense Feat.

### ISSUE THREAD

**Issue by Elvenoob** [openlegend/core-rules#105]
Now that the Armor bonus has been nerfed quite a bit, I noticed the unarmored fighting style users suffer substantially when it comes to those steep pre-requisites for the Natural Defense Feat, even before the update, taking a 7 in an almost useless-feeling stat to get the highest tier of it seemed way too much, and now, with Armor only affecting one defense, I don't think any character would actually take the final tier of this feat.

**Comment by Elvenoob** [comment]
(Particularly since, you can take Fleet of Foot for 2 feat points, and with a Fortitude of 4, you are now at the exact same +6 Evasion as when you MAX OUT this feat, but, for much lower cost.)

I'd say this needs another tier, and far lower pre-requisites across the board.

**Comment by raihje** [comment]
The benefits of Natural Armour are always active, as opposed to wearing said armour + afaik it can't be 'sundered' (we need a sunder armour bane..)
However, the requisite should definitely be lowered like 1 point across the board.
Fort 2/4/6 with 2/4/6 feat points required.

**Comment by Elvenoob** [comment]
Since such a bane doesn't exist yet, and might not end up being created, though. I feel Armor Mastery II also means, the defensive boost is always-on, because you can sleep in it, and the movement reduction is removed for heavier armors, meaning someone going Scale Mail gets a +8, for 4 feat points, no downside currently.

It can be nerfed back a bit if an Armor-destroying bane is made, but for now, I think it needs to be set to compete with Armor Mastery, at least something like +6 for 4 feat points. (Maybe in 2 tiers costing 2 and giving a +3 bonus, Tier 1 requiring a Fortitude of 2, Tier 2 requiring a 4.)

**Comment by raihje** [comment]
That'd be too easy for people, everyone would get 4 for by lv2 for free armour for the rest of the game. At least with armour mastery they still need to buy armour (you only start with wealth 2) and all that entails before they even worry about having it stolen from them.

**Comment by Elvenoob** [comment]
You get to start with 3 wealth 2 items, for a melee combatant character this will tend to look like:

- Armor
- Weapon
- ???

Owning weapons/armor is really not a big issue IMO, and again, if they're wearing it literally all the time, except during baths, it would be rather forced to somehow have that stolen during that tiny window. Even then, they can kill the thief and get it back, or just buy a new set since it's still Wealth 2.

(And if they're Armor Mastery-ing, they don't take speed penalties either. Nor would they "upgrade" to a "better" armor because they've specialized into the one they're using.)

Basically, it's just minor inconveniences that you're doing away with.

Don't get me wrong, I do _kinda_ like your idea, but... No-one is ever going to choose tier 1 on it's own, ever, if the bonus given is just 2. They might as well just throw on one of the Fort 2 armors, they'll get better Evasion for it.

**Comment by brianfeister** [comment]
**Natural Defense** synergizes with Berserker intentionally, you don't get the **Berserker** defense bonus if you're wearing armor.

**Comment by Elvenoob** [comment]
Ok, one specific character type, and they're the ones who get to play with the Fatigue bane. I'm not really all that concerned with them. Because Fatigue.

But, if the feat is so weak that the only reason to take it is in conjunction with the Berserker lines, what's the point of even having it?

I think, it needs some kind of significant buff, particularly since it now only affects one defense.

**Comment by brianfeister** [comment]
I agree with you. I'm not sure what that tweak should be exactly, so let's focus the conversation on the proposed changes and the merits of each.

**Comment by Elvenoob** [comment]
Hmm... I think it would be helpful to know why the Fortitude pre-requisites are there in the first place,  because that was always something which baffled me, particularly when it comes to the characters like Monks, who wouldn't really have an actual high fortitude. 

(although, I could see Attribute Substitution (Will -> Fortitude) working really well.)

Anyway, yeah, I think understanding why it was originally made the way it was will probably help.

(Because, right now, my line of thought is more leaning towards scrapping the Pre-requisites entirely)

**Comment by raihje** [comment]
I can see the point of not wanting 2 armour for 2 feat points, because leather armour is cheap.
So 3 / 6 / 9 is reasonable I guess, at 3 / 5 / 7 or 2 / 4 / 6 fort.
This way it's pretty strong, but by keeping the fort req. you make it still synergize with strong melee characters who have modelled their bodily training on their sheet.As for Martial characters it's typically 2 weapons and armour, or a weapon and shield with armour. I don't understand why your so afraid of detrimental effects like Fatigue. The first level hardly even does anything. The second one can easily be circumvented. You make Fatigue out to be much worse than it is - so much so you'll avoid any play styles that would result in Fatigue. At any rate, 3 / 6 / 9 armour for 3 / 5 / 7 doesn't sound too bad. If we lower it to 2 / 4 / 6 you'll have some of the best (I'd have to do the math but it's damn good) defense in the game at 6th level, which isn't too bad, but 3 / 5 / 7 is a smoother progression for people to invest in. Most fighters will have high agility or might, with decent fortitude and/or will anyway for the hp.

**Comment by raihje** [comment]
So a bit of basic comparing here.
The most min-maxed possible armour is elven plate. But lets just do a quick comparison of our
level 1 options. Might can be - 1-3 and speed can be - 5-10 (tier 2/3)

| Armour    | Might    | Speed P. | AC  |
|----------------|:----------:|:-----------:|------:|
|Leather       | 0           | 0              |2-5   |
|Scale Shirt  | 2-0        | 0              |5-7   |
|Scale Mail   | 4-1        | 5'-0'         |7-9  |
After lv 1 or 2 when you get a wealth level:
|Plate Mail   | 5-2         | 10'-0'      |8-10 |

So reducing the requirements to 2/4/6 isn't so unreasonable with the above comparison.
So 3 / 6 / 9 Armour for 2 / 4 / 6 Fortitude. It seems cheesy to me, but it only works while not wearing armour so it fundamentally requires a choice. You can be "competitive" and even completely out class lower armours without armour, or you can specialize in a particular armour (and the risks/costs associated) to get an edge faster (or an edge over *everyone* later with plate mail), I'm not sure how I feel about skin being tougher than base plate mail, but whatever. This gives players an incentive to use either feat.

**Comment by brianfeister** [comment]
I think this higher bump at Tier 1 from @raihje might make sense. It does of course incentivize taking the feat only one time, although I suppose it's OK because if you decide to start wearing armor, the usage of the feat is discarded ... hmmm ... seems like it might work.

**Comment by raihje** [comment]
So I'm fine to implement 3/5/7 for the armour bonus? Leave the Fort. requirement as is? (3/5/7) Or lower it to 2/4/6?

**Comment by brianfeister** [comment]
I think Fort should become Might

**Comment by brianfeister** [comment]
but leave as 3/5/7

**Comment by raihje** [comment]
Wouldn't fortitude make more sense, since it's essentially your bodily ability to withstand harm? I mean, I can put it in as Might instead, I guess.

**Comment by brianfeister** [comment]
Ok sure, I'm fine with Fort

**Comment by Bobiobobio** [comment]
I like the idea of the bonus and the prerequisite being equal. It makes you feel as though the feat makes your fort score apply to your armor score.

**Comment by raihje** [comment]
I just got a hard drive for my desktop and in the process of migrating, I'll tackle this in the morning.
